### DISCUSSION TITLE
Web3Auth v3 Release

### DISCUSSION THREAD

**Discussion by shahbaz17** [Web3Auth/Web3Auth#743] [Announcements]
Hey Web3Auth Community 👋,

Today, we have made a major release for [Web3Auth](https://web3auth.io/). We have taken a few steps in our latest release to streamline our repositories and package names and avoid confusion within the community. 

The latest release for [`Web3Auth v3`](https://github.com/Web3Auth/web3auth-web/releases) contains the following changes:
- React 18 Support
- New Ethereum deps used (lighter builds, better support for post-merge tx)
- Multi-locale support to the Login Modal. [See Screenshot](https://user-images.githubusercontent.com/6962565/196881887-75548bea-b54a-4094-90bc-d2fda7b291a4.png)
- [@web3auth/web3auth](https://www.npmjs.com/package/@web3auth/web3auth) is now [@web3auth/modal](https://www.npmjs.com/package/@web3auth/modal) 

This also comes with a major naming convention. All the packages under `@web3auth` will refer to Web3Auth Plug and Play.

If you’re looking to Web3Auth Self Host, packages under [`@tkey`](https://github.com/tkey) and [`@toruslabs`](https://github.com/torusresearch) support that.

Also, for our repositories, we have renamed our [`Web3Auth`](https://github.com/web3Auth/web3auth) repository to [`web3auth-web`](https://github.com/web3Auth/web3auth-web), which currently houses the discussions.

**Comment by aristidesgpg** [comment]
@shahbaz17 When using @tkey, could you tell me how we can handle the refresh?
So after login using tkey, if refresh the page, I want to keep the login

**Reply by yashovardhan** [reply]
Hey @talentdev2020 can you ask this question in a separate thread with code snippets and explaining your issue in detail?

**Comment by chusla** [comment]
friendly reminder to add french please!  Thank you!

https://github.com/orgs/Web3Auth/discussions/562
